package.yaml

```diff
@@ -0,0 +1,42 @@
+name:                dhall-kubernetes
+version:             0.1.0.0
```

Comment:
I'd prefer if we just use  the cabal file and not use `package.yaml`.     Seems kind of weird to use `yaml` in a repo meant to defeat the usage of yaml :P 

If you really can't stand `.cabal` file format we can perhaps have  a look at https://github.com/dhall-lang/dhall-to-cabal

But I guess this is just here because the default stack template uses this

Reply:
Indeed, I just went with a random template, I planned to remove this. I'll add it to the TODO list

Reply:
Removed this and added version bounds!